playwright/.auth/user.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
hm I thought this was usually gitignore'd?

Reply:
huh yeah, i will remove it